packages/nova-react/src/eventing/react-event-source-mapper.ts

```diff
@@ -0,0 +1,85 @@
+import { EventWrapper, InputType, Source } from "@nova/types";
+import { ReactEventWrapper } from "./nova-eventing-provider";
+
+/**
+ * Maps DOM event type strings to the Nova InputType.
+ *
+ * Initial implementation is not exhaustive and should be
+ * extended as new requirements arise.
+ *
+ * Note following excluded event types that could be considered:
+ * scroll, input, submit, toggle
+ */
+const typeMap = {
+  dblclick: InputType.mouse,
+  keydown: InputType.keyboard,
+  keyup: InputType.keyboard,
+  keypress: InputType.keyboard,
+  mousedown: InputType.mouse,
+  mouseenter: InputType.mouse,
+  mouseleave: InputType.mouse,
+  mousemove: InputType.mouse,
+  mouseover: InputType.mouse,
+  mouseout: InputType.mouse,
+  mouseup: InputType.mouse,
+  touchcancel: InputType.touch,
```

Comment:
it would be good to understand if pen events manifest as touch as well. But I don't have a pen device :(

Reply:
Tried it on Surface laptop and pen event is recognized correctly.
<img width="461" alt="image" src="https://user-images.githubusercontent.com/13124780/158185937-5e186b92-7582-4e23-ab83-9322a6ac87bd.png">